pages/management/SettingsHandler.inc.php

```diff
@@ -47,6 +47,7 @@ function settings($args, $request) {
 			case 'index':
 			case '':
 			case 'journal':
```

Comment:
I would suggest renaming the endpoint to `context` instead of `journal`/`press` throughout -- no sense in maintaining both, and having two working endpoints for the same place will probably lead to inadvertent breakage later.